workspaces/liferay-jethr0-workspace/client-extensions/liferay-jethr0-custom-element/src/pages/ProjectPage/ProjectPage.jsx

```diff
@@ -48,9 +48,9 @@ function ProjectPage() {
 	}
 
 	const breadcrumbs = [
-		{ active: false, link: '/', name: 'Home' },
-		{ active: false, link: '/projects', name: 'Projects' },
-		{ active: true, link: '/projects/{id}', name: projectName },
+		{active: false, link: '/', name: 'Home'},
+		{active: false, link: '/projects', name: 'Projects'},
+		{active: true, link: '/projects/{id}', name: projectName},
```

Comment:
@michaelhashimoto I think the reason these SF changes didn't get picked up is because you have `*.jsx` extension for your files when it should be `*.js`.

Do these have to be jsx? If so, we can update our tool